site/public/css/polls.css

```diff
@@ -39,10 +39,28 @@
     cursor: pointer;
 }
 
-.response-btn {
+.option-btn {
+    vertical-align: middle;
+}
+
+.option-btn input[type="radio"]{
     vertical-align: top;
 }
 
+.option-btn input[type="checkbox"]{
+    transform: scale(0.8);
+}
+
+.student-submit[type="submit"] {
+    margin-top: 2em;
+}
+
+.option-text {
+    padding-top: 0.55em;
+    padding-bottom: 0.55em;
+    padding-left: 0.25em;
+}
```

Comment:
same here

Reply:
Yes, I tried to keep as little CSS as possible and also replicate its original look. This does make the text cell look nice with alignment, spacing, and stuff.

Reply:
That seems reasonable.  Hundredths of `em`s are very small values, so you might just stick with `0.5em` for the top and bottom, but that's just my personal opinion.  I'll approve it either way.

Reply:
Makes sense.